strr-base-web/app/app.config.ts

```diff
@@ -52,8 +52,8 @@ export default defineAppConfig({
       label: '-mt-[2px] pl-3 text-base cursor-pointer'
     },
     radioGroup: {
-      wrapper: 'max-w-bcGovInput',
-      fieldset: 'space-y-4 -mt-4',
+      wrapper: 'max-w-bcGovInput group',
```

Comment:
adding space-y-4 included the legend, even when legend is sr-only, had to apply negative margin to fix this scenaria